Tests.Unit.Presentation.Web/Services/ReferenceServiceTest.cs

```diff
@@ -432,61 +432,103 @@ public void AddReference_Does_Not_Save_Root_If_Add_Fails()
         }
 
         [Fact]
-        public void Can_BatchUpdateExternalReferences_With_No_Previous_References()
+        public void Can_CreateExternalReferences()
```

Comment:
no reference to the method here,,,

Reply:
naming refers to the update case but validates a specific effect... naming should be better